### PR TITLE
ovn: remove 'func (oc *Controller)' from functions that don't need it

### PR DESCRIPTION
**Pull request by dcbw**
@danwinship @girishmg

### CODE REVIEW

File: go-controller/pkg/ovn/policy.go [hunk 1]
```diff
@@ -33,20 +32,18 @@ func (oc *Controller) syncNetworkPoliciesPortGroup(
...
 	})
 	if err != nil {
 		logrus.Errorf("Error in syncing network policies: %v", err)
 	}
 }
 
-func (oc *Controller) addACLAllow(np *namespacePolicy,
-	match, l4Match string, ipBlockCidr bool, gressNum int,
-	policyType knet.PolicyType) {
+func addACLAllow(np *namespacePolicy, match, l4Match string, ipBlockCidr bool, gressNum int, policyType knet.PolicyType) {
```
Comment (danwinship) [inline go-controller/pkg/ovn/policy.go]: Do we like long lines now? (I do...)

File: go-controller/pkg/ovn/policy_common.go [hunk 2]
```diff
@@ -30,6 +30,19 @@ type namespacePolicy struct {
 	deleted         bool //deleted policy
 }
 
+func NewNamespacePolicy(policy *knet.NetworkPolicy) *namespacePolicy {
```
Comment (danwinship) [inline go-controller/pkg/ovn/policy_common.go]: not really related to the rest of the PR...

File: go-controller/pkg/ovn/policy.go (hunk 1 above)
Comment (dcbw) [reply go-controller/pkg/ovn/policy.go]: @danwinship I do, as long as they aren't horribly long... I try to keep under like 90 or 100. Worth a try :)

File: go-controller/pkg/ovn/policy_common.go (hunk 2 above)
Comment (dcbw) [reply go-controller/pkg/ovn/policy_common.go]: Ok.

**Pull request merged**